packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts

```diff
@@ -149,7 +149,9 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
                 // tslint:disable-next-line: no-any
                 (compilation.mainTemplate.hooks as any).assetPath.tap(
                   'build-angular',
-                  (filename: string, data: ChunkData) => {
+                // tslint:disable-next-line: no-any
+                  (filename: any, data: ChunkData) => {
+                    if (typeof filename === 'function') { filename = filename(data); }
```

Comment:
Can you rework this so it does not reassign the argument?

Reply:
Sure